kolona/task.py

```diff
@@ -13,6 +15,7 @@ class GlobalTask:
     max_retries: int
     retry_intervals: List[int] = [3, 5, 15]
     last_attempt_time = 0
+    default_delay: Seconds = 0
```

Comment:
I think we should name this `delay`. One reason is that all existing attributes are named without `default_` the other is from the perspective of how it would be used and I think it makes the code a bit weird, eg:

```py
@task(queue=queue, default_delay=5)
def super_duper_task():
    pass
```

I think just passing in `delay` makes sense here, similarly as we don't pass in `default_queue` despite we allow overriding a queue when calling `enqueue()`. I think it makes it obvious that the values set on a decorator are the defaults that are then used when calling `enqueue()` and that any attribute passed into `enqueue()` would override the default value for that particular "enqueuement"?

```py
@task(queue=queue, delay=5)
def super_duper_task():
    pass
```

I understand you noted this when you first opened the PR, it was my fault I didn't really notice it before.